gnocchi/tests/functional/gabbits/metric-list.yaml

```diff
@@ -131,12 +125,18 @@ tests:
           $[1].archive_policy.name: first_archive
           $[2].archive_policy.name: first_archive
 
-    - name: list metrics by user_id
-      GET: /v1/metric?user_id=faf30294217c4e1a91387d9c8f1fb1fb
+    - name: list metrics by creator jd
+      GET: /v1/metric?creator=jd
+      request_headers:
+        # User admin
+        authorization: "basic YWRtaW46"
       response_json_paths:
           $.`len`: 1
 
-    - name: list metrics by project_id
-      GET: /v1/metric?project_id=f3ca498a61c84422b953133adb71cff8
+    - name: list metrics by creator foobaz
+      GET: /v1/metric?creator=foobaz
+      request_headers:
+        # User admin
+        authorization: "basic YWRtaW46"
       response_json_paths:
-          $.`len`: 2
+          $.`len`: 1
```

Comment:
Because the test was previously checking by `created_by_project_id`, which is always empty when using basic auth (remember, this field is legacy). So I change the test to a different `creator` test filtering instead.